core-web/apps/dotcms-ui/src/app/portlets/dot-categories/dot-categories-list/dot-categories-list.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component, EventEmitter, Output, ViewChild } from '@angular/core';
+import { DotListingDataTableComponent } from '@components/dot-listing-data-table/dot-listing-data-table.component';
+import { Observable } from 'rxjs';
+import { DotCategoriesListStore, DotCategoriesListState } from './store/dot-categories-list-store';
+import { DotCategory } from '@dotcms/app/shared/models/categories/dot-categories.model';
+import { take } from 'rxjs/operators';
+import { MenuItem } from 'primeng/api/menuitem';
+@Component({
+    selector: 'dot-categories-list',
+    templateUrl: './dot-categories-list.component.html',
+    styleUrls: ['./dot-categories-list.component.scss'],
+    providers: [DotCategoriesListStore]
+})
+export class DotCategoriesListComponent {
+    vm$: Observable<DotCategoriesListState> = this.store.vm$;
+    @Output() updateCategory: EventEmitter<MenuItem> = new EventEmitter();
+    @ViewChild('listing', { static: false })
+    listing: DotListingDataTableComponent;
+
+    constructor(private store: DotCategoriesListStore) {}
+
+    /**
+     * The function clears the global search of listing-data-table by calling the clearGlobalSearch() function on the listing
+     * object
+     */
+    clearState() {
+        this.listing.clearGlobalSearch();
+    }
+
+    /**
+     * It set a selected category to the parent component, updates the endpoint for the listing component,
+     * and then loads the first page of the listing component
+     * @param {DotCategory} category - DotCategory - The category object that is being passed to the
+     * function.
+     */
+    addBreadCrumb(category: DotCategory) {
+        const getSubCategoryEndPoint = 'v1/categories/children';
+        this.store.updateCategoryEndPoint(getSubCategoryEndPoint);
+        this.store.addCategoriesBreadCrumb({ label: category.categoryName, id: category.inode });
```

Comment:
Don't update the state twice when no need it.

Reply:
updated